model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/solve_nonhydro.py

```diff
@@ -389,9 +389,6 @@ def _validate(self):
         if self.divdamp_order != DivergenceDampingOrder.COMBINED:
             raise NotImplementedError("divdamp_order can only be 24")
 
-        if self.divdamp_type == DivergenceDampingType.COMBINED:
-            raise NotImplementedError("divdamp_type with value 32 not yet implemented")
-
```

Comment:
I think it is safe to remove that as the ICON code only ever checks for divdamp_type >=3, so inside solve_non_hydro `COMBINED` and `THREE_DIMENSIONAL` are identical. But there is also the option `TWO_DIMENSIONAL = 2` which does not exist at all in `solve_nonhydro.py`. Can you add this to the enum and change the above lines to instead of deleting them?
```
if self.divdamp_type == DivergenceDampingType.TWO_DIMENSIONAL:
            raise NotImplementedError("`DivergenceDampingType.TWO_DIMENSIONAL` (2) is not yet implemented")
```